data/funds.json

```diff
@@ -25,6 +25,58 @@
         "factsheet": "https://global.vanguard.com/portal/site/loadPDF?country=nl&docId=31658",
         "index": "FTSE All-World Index"
     },
+    {
+        "name": "Vanguard FTSE Developed Europe UCITS ETF (Accumulating)",
+        "type": "ETF",
+        "symbol": "VWCG",
+        "isin": "IE00BK5BQX27",
+        "logo": "vanguard.png",
+        "totalExpenseRatio": 0.10,
+        "internalTransactionCosts": 0.015,
+        "dividendLeakage": 11.42,
```

Comment:
De dividendleakage hier los per fonds zetten zal helaas niet tot een juiste totaal berekening leiden.

De hoogte van het dividend verschil per regio. In Amerika ligt dividend bijvoorbeeld vrij laag. Het dichten van het lek daar heeft dan net wat minder effect als conform het market cap gewicht.

Om dit op te lossen heb ik zelf gewoon een lek van 4,8% of 5% overall over deze portefuille genomen, dan kom je wel goed uit.

Ik snap dat dit lastig in het standaard model hier past. Ik heb ook wel een excel met historische dividend hoogte per regio. Dan zou je dit cijfer daarmee daar wat op kunnen corrigeren. Maar dat maakt het wel ingewikkeld. 5% voor de hele portefeuille pakken is veel eenvoudiger.

Reply:
Jouw voorstel dus om alle fondsen in dit portfolio op 5% te zetten? Lijkt me wel een beetje smerige oplossing en voor problemen gaan zorgen als je die fondsen zou hergebruiken in andere portfolio's (bv zonder invesco en dus dat gat niet gedicht hebt).

Waarom zou het niet goed gaan als alle fondsen de juiste corresponderende percentages gezet krijgen die binnen het portfolio naar cap ratio meegeteld worden?

Reply:
Voorbeeld: Stel USA is 50% van de wereld in cap en heeft 0% lekkage via een fonds. De rest is de andere 50% en heeft 12% lekkage. De totale lekkage is dan **niet** simpelweg 6%.

Waarom: USA keert gemiddeld maar zo'n 2% dividend uit, de rest van de wereld richting de 3%. Ofwel: de 0% van de USA weegt minder zwaar als simpelweg 50% omdat er minder dividend wordt uitgekeerd.

5% lek over het geheel geven is het meest eenvoudig en correct, niks mis mee. Ja daarmee is het niet herbruikbaar.

Als je dat wel wil zul je correcties per regio moeten doen op het lek aan de hand van historisch dividend hoogte in die regio. Ik heb daar wel data voor wat de dividend hoogte per regio was over de afgelopen 5 jaar.  Maar ik ga niet kunnen beloven dat ik die data jaarlijks up to date ga houden ;-) Je kan die data wel gewoon zelf uit de MSCI database peuteren.

Reply:
Als de reden voor incorrectie van de dividendberekening o.b.v. percentages en cap ratio komt uit het feit dat USA doorgaans minder dividend uitkeert is de afwijking in de berekening toch eigenlijk al gevangen in de disclaimer:
> Bovendien kunnen de rendementen van fondsen onderling verschillen door het volgen van een andere index of door het toepassen van verschillende uitsluitingscriteria
Er is immers geen mogelijkheid om rendement/dividend in te stellen per fonds.

Ik bereken op basis van caps en de huidige dividend percentages (weet niet of die correct zijn) 4,5% zonder IUSN en 5,4% met. Close enough?

Reply:
> Close enough?

Sorry dit is niet mijn mentaliteit zoals je misschien weet 😄 

Ander / simpeler idee: pak de huidige dividendpercentages uit de factsheet van de indexen en reken daarmee. Pas het lekkage naar boven en naar beneden aan op basis van hoe hoog dividend in die regio is.

Voorbeeld op wat voor lek je dan uit komt:

[regio-lek-git.xlsx](https://github.com/nicwortel/indexfondsenvergelijken.nl/files/10527465/regio-lek-git.xlsx)

De "Yield corrected dividend leakage" moet je dan pakken. Daar is dan wel het feit dat dividend sterk per regio verschilt meegewogen.